lib/ui/views/login_view.dart

```diff
@@ -54,7 +60,16 @@ class _LoginViewState extends State<LoginView> {
                               Hero(
                                 tag: 'ets_logo',
                                 child: Image.asset(
-                                  'assets/images/ets_white_logo.png',
+                                  model.getAppTheme() ==
+                                          ThemeMode.light.toString()
+                                      ? "assets/images/ets_white_logo.png"
+                                      : model.getAppTheme() ==
+                                              ThemeMode.dark.toString()
+                                          ? "assets/images/ets_red_logo.png"
+                                          : Theme.of(context).brightness ==
+                                                  Brightness.light
+                                              ? "assets/images/ets_white_logo.png"
+                                              : "assets/images/ets_red_logo.png",
```

Comment:
Same here. Did you found a case where this is needed? 🤔 
```suggestion
                                  Theme.of(context).brightness ==
                                                  Brightness.light
                                              ? "assets/images/ets_white_logo.png"
                                              : "assets/images/ets_red_logo.png",
```